src/dump_analysis.cpp

```diff
@@ -737,15 +737,12 @@ static void anal_dump_struct_field(AnalDumpCtx *ctx, const TypeStructField *stru
 
     jw_begin_object(jw);
 
-    jw_object_field(jw, "name");
-    jw_string(jw, buf_ptr(struct_field->name));
+    jw_object_field(jw, "src");
+    anal_dump_node_ref(ctx, struct_field->decl_node);
```

Comment:
We can avoid data duplication here by omitting this field here, and instead emitting it in `anal_dump_node`, by detecting NodeTypeContainerDecl, and outputting the array of field nodes.

Reply:
This is now done for struct fields.